db/migrate/20170515061739_add_more_options_to_course_assessments.rb

```diff
@@ -0,0 +1,10 @@
+class AddMoreOptionsToCourseAssessments < ActiveRecord::Migration
+  def change
+    add_column :course_assessments, :show_private, :boolean,
+               default: false,
+               comment: 'Show private test cases after students get the answer correct'
```

Comment:
Small issue - `Show private test cases after students answer correctly` - Can change for the other comment as well?